bundles/org.openhab.binding.intesis/src/main/java/org/openhab/binding/intesis/internal/IntesisHandlerFactory.java

```diff
@@ -73,6 +77,11 @@ public boolean supportsThingType(ThingTypeUID thingTypeUID) {
             return new IntesisHomeHandler(thing, httpClient, intesisStateDescriptionProvider);
         }
 
+        if (THING_TYPE_INTESISBOX.equals(thingTypeUID)) {
+            logger.debug("Creating a IntesisBoxHandler for thing '{}'", thing.getUID());
```

Comment:
This is logged by the framework an can therefore be removed.

Reply:
removed